content/TruenasApps/UsingCustomApp.md

```diff
@@ -48,138 +35,1199 @@ If you are unsure about any configuration settings, review the [Install Custom A
+
+   Enter in **Tmpfs Size Limit (in Mi)** the maximum size of the temporary directory in mebibytes or accept the default of *500*.
+   {{< /expand >}}
+
+8. {{< include file="/static/includes/apps/AppInstallWizardResourceConfig.md" >}}
+
+9. Use **GPU Configuration** to allocate GPU resources if available and required for the application.
+
+10. Click **Install** to deploy the container.
     If you correctly configured the app, the widget displays on the **Installed Applications** screen.
 
-    When complete, the container becomes active. If the container does not automatically start, click **Start** on the widget.
+    When complete, the container becomes active.
+    If the container does not automatically start, click **Start** on the widget.
 
     Click on the App card reveals details.
 
-## Setting up App Storage
+## Installing via YAML
 
-### Defining Host Path Volumes
+{{< include file="/static/includes/apps/YAMLWarning.md" >}}
 
-You can mount TrueNAs storage locations inside the container.
-To mount TrueNAS storage, define the path to the system storage and the container internal path for the system storage location to appear.
-You can also mount the storage as read-only to prevent using the container to change any stored data.
-For more details, see the [Kubernetes hostPath documentation](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath).
+To install a third-party application using a Docker Compose YAML file, go to **Apps > Discover**.
+Click <i class="material-icons" aria-hidden="true" title="more_vert">more_vert</i> then select **Install via YAML** to open the **Add Custom App** screen.
+
+{{< trueimage src="/images/SCALE/Apps/InstallCustomAppYAML.png" alt="Install Custom App via YAML" id="Install Custom App via YAML" >}}
+
+The **Add Custom App** screen contains two fields.
+
+Enter a name for the application in **Name** using lowercase, alphanumeric characters.
+
+Enter the Compose YAML file in **Custom Config**.
+
+Ensure the Compose file begins with a top-level element, such as `name:` or `services:`.
+See the Docker [Compose file reference](https://docs.docker.com/reference/compose-file/) for more information.
+
+As Compose files can be complex and YAML relies on indentation and whitespace to define structure and hierarchy, we recommend writing the file in a stand-alone code editor before pasting the completed content into the **Custom Config** field.
+
+Click **Save** to begin deployment of the app.
+
+### YAML Deployment Examples
+
+The following examples represent some of the capabilities of Docker Compose and how it can be used to configure applications in TrueNAS.
+This is not an exhaustive collection of all capabilities, nor are the files below intended as production-ready deployment templates.
+
+Note that app storage in these examples is configured using [volumes](https://docs.docker.com/engine/storage/#volumes), which are managed by the Docker engine.
```

Comment:
```suggestion
Note that app storage in these examples are configured using [volumes](https://docs.docker.com/engine/storage/#volumes), which are managed by the Docker engine.
```

Reply:
The noun here is "storage", so "is" agrees with the noun.